scripts/jenkins/jobs-obs/openstack-trackupstream.yaml

```diff
@@ -129,7 +133,10 @@
               "openstack-nova-virt-zvm",
               "openstack-ec2-api",
               "python-networking-cisco",
-              "python-networking-hyperv" ].contains(component) )
+              "python-networking-hyperv" ].contains(component) ||
+            [ "Cloud:OpenStack:Liberty:Staging", "Cloud:OpenStack:Kilo:Staging", "Cloud:OpenStack:Juno:Staging" ].contains(project) &&
+            [ "openstack-magnum",
+              "openstack-horizon-plugin-magnum-ui" ].contains(component) )
```

Comment:
@dirkmueller why? do we want to track magnum for Liberty, too? I thought it's >=Mitaka only...